modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Testflow/Subtask/SubtaskCaseResults.tsx

```diff
@@ -20,13 +20,58 @@ import StatusBadge from '../../../components/StatusBadge';
 import {StatusBadgeType} from '../../../components/StatusBadge/StatusBadge';
 import {ListViewTypes} from '../../../context/ListViewContext';
 import i18n from '../../../i18n';
+import {Liferay} from '../../../services/liferay';
 import {TestraySubTaskCaseResult} from '../../../services/rest';
 import {testraySubtaskCaseResultImpl} from '../../../services/rest/TestraySubtaskCaseResults';
 import {searchUtil} from '../../../util/search';
 
 const SubtasksCaseResults = () => {
 	const {subtaskId} = useParams();
 
+	const getFloatingBoxAlerts = (
+		subtasksCaseResults: TestraySubTaskCaseResult[],
+		selectRows: number[]
+	) => {
+		const alerts = [];
+
+		if (subtasksCaseResults.length === selectRows.length) {
+			alerts.push({
+				text: i18n.translate(
+					'you-cannot-split-all-case-results-from-a-subtask'
+				),
+			});
+		}
+
+		const userSubtaskCheck = () => {
```

Comment:
```suggestion
		const subtaskUserCheck = () => {
```

Reply:
just because of the translation that this way it is better to translate into Portuguese